package.json

```diff
@@ -48,10 +49,13 @@
     "@semantic-release/github": "^7.2.0",
     "@semantic-release/npm": "^7.0.10",
     "@semantic-release/release-notes-generator": "^9.0.2",
-    "@types/jasmine": "^3.6.6",
     "atom-ts-transpiler": "^1.5.3",
     "eslint": "^7.22.0",
     "semantic-release": "^17.4.2",
     "typescript": "^4.2.3"
+  },
+  "optionalDependencies": {
```

Comment:
What is the benefit of using `optionalDependencies` here?

Reply:
If people want to use the types for this package, they need to install the types for Atom and Jasmine